src/main/java/com/iexec/core/worker/WorkerService.java

```diff
@@ -191,6 +205,8 @@ public int getAliveAvailableGpu () {
 
     // region Read-and-write methods
     public Worker addWorker(Worker worker) {
+        workerStatsMap.computeIfAbsent(worker.getWalletAddress(), WorkerStats::new)
+                .setLastAliveDate(new Date());
```

Comment:
Should we instead call `updateLastAlive`?

Reply:
see a1ad622